views/editQuiz.handlebars

```diff
@@ -0,0 +1,53 @@
+<head>
+    <meta charset="utf-8"/>
+    <title>Quizicle</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" crossorigin="anonymous">
+    <link rel="stylesheet" href="/style.css">
+</head>
+
+<body>
+    {{> header}}
+
+    <div class="edit_quiz">
+        <div class="intro">
+	    <div class="edit_title">
+	        <input id="edit-quiz-title-input" placeholder="Title..."></input>
+	    </div>
+	    <div class="edit_description">
+	        <textarea id="edit-quiz-description-input" placeholder="Description..."></textarea>
+	    </div>
+	    <div class="edit-quiz-tags">
```

Comment:
All class names need to use `snake_case` with underscores. Please fix all the variable names to be consistent